src/components/StatusBar.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+function StatusBarTail({img,title, count }) {
+    return (
+          <div className="justify-center items-center">
```

Comment:
Why so many divs ?
Can we not combine into a single div ?

Reply:
reduced as much as possible